src/Qwiq.Core.Rest/Project.cs

```diff
@@ -64,7 +68,27 @@ internal Project([NotNull] TeamProjectReference project, [NotNull] WorkItemStore
                                               .GetResult();
 
                             return new Node(result).ChildNodes;
-                        }))
+                        }),
+                new Lazy<IQueryFolderCollection>(() =>
+                {
+                    return new QueryFolderCollection(() =>
+                    {
+                        //BUGBUG: There is a bug in the GetQueryAsync in vsts where if a folder contains a '+' character it will return 404, even if the folder exists see here: https://developercommunity.visualstudio.com/content/problem/123660/when-a-saved-query-folder-contains-a-character-it.html
+                        QueryHierarchyItem FolderExpansionFunc(QueryHierarchyItem item)
+                        {
+                            try
+                            {
+                                return store.NativeWorkItemStore.Value.GetQueryAsync(project.Id, item.Path, QueryExpand.Wiql, MaxQueryFolderExpansionDepth).Result;
```

Comment:
Please review method for awaiting for and obtaining result (s.g. L67 L68)

Reply:
Changed to GetAwaiter().GetResult